ArduPlane/is_flying.cpp

```diff
@@ -18,7 +18,7 @@ void Plane::update_is_flying_5Hz(void)
     bool is_flying_bool = false;
     uint32_t now_ms = AP_HAL::millis();
 
-    uint32_t ground_speed_thresh_cm = (aparm.min_gndspeed_cm > 0) ? ((uint32_t)(aparm.min_gndspeed_cm*0.9f)) : GPS_IS_FLYING_SPEED_CMS;
+    uint32_t ground_speed_thresh_cm = (aparm.min_groundspeed > 0) ? ((uint32_t)(aparm.min_groundspeed*90)) : GPS_IS_FLYING_SPEED_CMS;
```

Comment:
Maybe a comment, or break out the `*100*0.9`. Just `*90` is not obvious.

Reply:
changed